<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN192431" failures="1" tests="11" name="Test" time="217.108" errors="0" timestamp="2024-02-07T22:24:43 IST">
  <testcase classname="testCase.TC_001_SearchingCardiologiests" name="SearchingCardiologiests" time="5.716"/>
  <testcase classname="testCase.TC_001_SearchingCardiologiests" name="gettingCardiologiests" time="20.584"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="CapturePopularSurgeries" time="8.998"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="GeneralSurgeries" time="4.191"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="OphthamologySurgeries" time="2.7"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="ProctologySurgeries" time="3.472"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="UrologySurgeries" time="4.01"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="CosmeticSurgeries" time="2.794"/>
  <testcase classname="testCase.TC_002_CaptureSurgeries" name="OrthopedicsSurgeries" time="5.142"/>
  <testcase classname="testCase.TC_003_CorporateHealthAndWellness_InvalidDetails" name="checkingInvalidDetails" time="13.606"/>
  <testcase classname="testCase.TC_004_CorporateHealthAndWellness_ValidDetails" name="checkingValidDetails" time="61.496">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCase.TC_004_CorporateHealthAndWellness_ValidDetails.checkingValidDetails(TC_004_CorporateHealthAndWellness_ValidDetails.java:106)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkingValidDetails -->
</testsuite> <!-- Test -->
